packages/core/src/components/tag/tag.tsx

```diff
@@ -73,6 +73,7 @@ export interface TagProps
  */
 export const Tag: React.FC<TagProps> = React.forwardRef((props, ref) => {
     const {
+        active: _active,
```

Comment:
Would wrapping `htmlProps` in `removeNonHTMLProps` achieve the same result?
```ts
{...removeNonHTMLProps(htmlProps)}
```

Reply:
I think so, will do that too to be safe